server/errors/index.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response, NextFunction } from 'express';
+import CustomError from './custom-error';
+
+const clientError = (req: Request, res: Response, next: NextFunction) => {
+  res.status(404).json({ message: 'Page Not Found' });
+};
+
+const serverError = (err: any, req: Request, res: Response, next: NextFunction) => {
+  res
+    .status(err.status || 500)
+    .json({ message: err.status ? err.message : 'Internal Server Error' });
```

Comment:
In my opinion, I believe that this should always be 'Internal Server Error', because in some time it might contain sensitive data especially if the error was returning from the database and this might tell the person who is trying to do something wrong what exactly is the problem with his approach and how to fix his mistake, and the developing team can always console log at the right places to see what is the problem without using this error message.